account_statement_base/models/account_journal_dashboard.py

```diff
@@ -0,0 +1,15 @@
+from odoo import models
+
+
+class AccountJournal(models.Model):
+    _inherit = "account.journal"
+
+    def create_cash_statement(self):
+        self.ensure_one()
```

Comment:
Please always document the why of the things:

```suggestion
        # Totally override this action for avoiding the standard
        # message saying that you need to install the enterprise
        # module. We do the equivalent thing instead.
        self.ensure_one()
```

Reply:
Thanks, I only added a similar comment in the commit message but not in the function itself. I will take that into account next time.